addons/website/static/src/js/show_password.js

```diff
@@ -0,0 +1,26 @@
+//
+// This file is meant to determine the timezone of a website visitor
+// If the visitor already exists, no need to keep the timezone cookie
+// as the timezone is set on the visitor.
+//
+odoo.define('website.show_password', function (require) {
+'use strict';
+
+var ajax = require('web.ajax');
+var utils = require('web.utils');
+var publicWidget = require('web.public.widget');
+
+publicWidget.registry.show_password = publicWidget.Widget.extend({
```

Comment:
Name it `ShowPassword` (CamelCase) as it is a class definition I started using CamelCase for those.